execute_unit_test.sh

```diff
@@ -3,4 +3,97 @@ echo "set the LIBQUICKJSC_TEST_PATH"
+flutter test test/datefmt/datefmt_es_US_test.dart
+flutter test test/datefmt/datefmt_et_EE_test.dart
+flutter test test/datefmt/datefmt_fa_IR_test.dart
+flutter test test/datefmt/datefmt_fi_FI_test.dart
+flutter test test/datefmt/datefmt_fr_CA_test.dart
+flutter test test/datefmt/datefmt_fr_FR_test.dart
+flutter test test/datefmt/datefmt_ga_IE_test.dart
+flutter test test/datefmt/datefmt_gu_IN_test.dart
+flutter test test/datefmt/datefmt_ha_Latn_NG_test.dart
+flutter test test/datefmt/datefmt_he_IL_test.dart
+flutter test test/datefmt/datefmt_hi_IN_test.dart
+flutter test test/datefmt/datefmt_hr_HR_test.dart
+flutter test test/datefmt/datefmt_id_ID_test.dart
+flutter test test/datefmt/datefmt_it_IT_test.dart
+flutter test test/datefmt/datefmt_ja_JP_test.dart
+flutter test test/datefmt/datefmt_ka_GE_test.dart
+flutter test test/datefmt/datefmt_kk_Cyrl_KZ_test.dart
+flutter test test/datefmt/datefmt_km_KH_test.dart
+flutter test test/datefmt/datefmt_kn_IN_test.dart
+flutter test test/datefmt/datefmt_ko_KR_test.dart
+flutter test test/datefmt/datefmt_ku_Arab_IQ_test.dart
+flutter test test/datefmt/datefmt_lt_LT_test.dart
+flutter test test/datefmt/datefmt_lv_LV_test.dart
+flutter test test/datefmt/datefmt_mk_MK_test.dart
+flutter test test/datefmt/datefmt_ml_IN_test.dart
+flutter test test/datefmt/datefmt_mn_Cyrl_MN_test.dart
+flutter test test/datefmt/datefmt_mr_IN_test.dart
+flutter test test/datefmt/datefmt_ms_MY_test.dart
+flutter test test/datefmt/datefmt_nb_NO_test.dart
+flutter test test/datefmt/datefmt_nl_NL_test.dart
+flutter test test/datefmt/datefmt_or_IN_test.dart
+flutter test test/datefmt/datefmt_pa_IN_test.dart
+flutter test test/datefmt/datefmt_pl_PL_test.dart
+flutter test test/datefmt/datefmt_pt_BR_test.dart
+flutter test test/datefmt/datefmt_pt_PT_test.dart
+flutter test test/datefmt/datefmt_ro_RO_test.dart
+flutter test test/datefmt/datefmt_ru_RU_test.dart
+flutter test test/datefmt/datefmt_si_LK_test.dart
+flutter test test/datefmt/datefmt_sk_SK_test.dart
+flutter test test/datefmt/datefmt_sl_SI_test.dart
+flutter test test/datefmt/datefmt_sq_AL_test.dart
+flutter test test/datefmt/datefmt_sr_Latn_RS_test.dart
+flutter test test/datefmt/datefmt_sv_SE_test.dart
+flutter test test/datefmt/datefmt_sw_KE_test.dart
+flutter test test/datefmt/datefmt_ta_IN_test.dart
+flutter test test/datefmt/datefmt_te_IN_test.dart
+flutter test test/datefmt/datefmt_th_TH_test.dart
+flutter test test/datefmt/datefmt_tr_TR_test.dart
+flutter test test/datefmt/datefmt_uk_UA_test.dart
+flutter test test/datefmt/datefmt_ur_IN_test.dart
+flutter test test/datefmt/datefmt_uz_Latn_UZ_test.dart
+flutter test test/datefmt/datefmt_vi_VN_test.dart
+flutter test test/datefmt/datefmt_zh_CN_test.dart
+flutter test test/datefmt/datefmt_zh_Hant_HK_test.dart
+flutter test test/datefmt/datefmt_zh_Hant_TW_test.dart
+flutter test test/datefmt/datefmt_WeekdayTranslation_test.dart
+flutter test test/datefmt/datefmt_MonthTranslation_test.dart
```

Comment:
There is no way to find and execute all of the tests automatically?

Reply:
I updated it as you suggested. Thank you!  ```find test -name '*_test.dart' | xargs -n 1 flutter test ```